Makefile

```diff
@@ -148,6 +148,14 @@ test-snuba: create-db
 		-vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml"
 	@echo ""
 
+test-snuba-ci: create-db
```

Comment:
can we remove the separate `snuba` suite in CI and run those as part of the normal suite? (in `backend.yml`) -- right now most of them are run twice -- in both suites

Reply:
Done. Renamed this to `test-snuba` then since that's the command Snuba expects. We can remove the required check from github after.